suites/create.js

```diff
@@ -308,6 +308,101 @@ export function runDataIntegrityProofFormatTests({
           }
         }
       });
+      describe('Algorithms', function() {
+        describe('Add Proof', function() {
```

Comment:
Do the nested describes show up properly in the HTML report?

Reply:
As far as I can tell they don't show up in the report, they are however useful when reading the test suite file and running the tests on the command line

Reply:
@BigBlueHat we would need to add a new feature to the reporter like `this.suite.subsection = true` and then collate and add those as subsections in the report. Feature should be proposed over on the reporter though and not here.

Reply:
Lets put that feature as a nice to have and postpone, the goal of the describe here was just to structure the tests when browsing the code and logically group the test cases